scripts/build_torch_wheels.sh

```diff
@@ -126,6 +126,7 @@ function install_llvm_clang() {
   sudo apt-get install -y clang-8 clang++-8
   maybe_append 'export CC=clang-8 CXX=clang++-8' ~/.bashrc
   export CC=clang-8 CXX=clang++-8
+  sudo update-alternatives --install /usr/bin/clang++ clang++ $(which clang++-8) 70
```

Comment:
Ugh, what happens if this image is used in the upstream, and PyTorch build uses a different clang version? If they try `ln -s` it will break.

Reply:
ehh, right. I think we need to add a check whether clang++ existed or not. I will revert this change as it will likely break pytorch upstream.

Reply:
@JackCaoG @will-cromar  sorry about the confusion, this is infra image, not circleci image